### ISSUE TITLE
Import's Instructions from Readme doesn't work

### ISSUE THREAD

**Issue by romanandreyvich** [touchlab/CrashKiOS#1]
I'm trying to use the imports instruction from readme:

```
kotlin {
   sourceSets {
       iosMain {
           dependencies {
               api "co.touchlab:crashkios:0.2"
           }
       } 
   }
}
```

The result is:

```
Execution failed for task ':multiplatform:compileKotlinIos'.
> Could not resolve all files for configuration ':multiplatform:iosCompileKlibraries'.
   > Could not find co.touchlab:crashkios:0.2.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/co/touchlab/crashkios/0.2/crashkios-0.2.module
       - https://dl.google.com/dl/android/maven2/co/touchlab/crashkios/0.2/crashkios-0.2.pom
       - https://jcenter.bintray.com/co/touchlab/crashkios/0.2/crashkios-0.2.module
       - https://jcenter.bintray.com/co/touchlab/crashkios/0.2/crashkios-0.2.pom
       - https://repo.maven.apache.org/maven2/co/touchlab/crashkios/0.2/crashkios-0.2.module
       - https://repo.maven.apache.org/maven2/co/touchlab/crashkios/0.2/crashkios-0.2.pom
       - https://jitpack.io/co/touchlab/crashkios/0.2/crashkios-0.2.module
       - https://jitpack.io/co/touchlab/crashkios/0.2/crashkios-0.2.pom
       - https://maven.google.com/co/touchlab/crashkios/0.2/crashkios-0.2.module
       - https://maven.google.com/co/touchlab/crashkios/0.2/crashkios-0.2.pom
       - https://kotlin.bintray.com/kotlinx/co/touchlab/crashkios/0.2/crashkios-0.2.module
       - https://kotlin.bintray.com/kotlinx/co/touchlab/crashkios/0.2/crashkios-0.2.pom
     Required by:
         project :multiplatform
```

I'm using Kotlin 1.3.60, Android Studio 3.5.2. Am I do something wrong?

**Comment by kpgalligan** [comment]
You are quite correct. Kind of rushed this out. Fixing.

**Comment by kpgalligan** [comment]
OK. That version was a local publish, which is why you weren't seeing it. I've published that properly, but it may take a bit to get through the caches. Will close when confirmed to be available.

**Comment by kpgalligan** [comment]
Fixed! New version publishing now, 0.2.1, but 0.2 is public.
